### ISSUE TITLE
Proposal: metadata tags <podcast:book>, <podcast:game>, <podcast:tv_show>

### ISSUE THREAD

**Issue by PofMagicfingers** [Podcastindex-org/podcast-namespace#180]
*This is a transfered proposal from our former project podCloud/podcast-ext. See #173 for details.*

### Episode metadata tags

Any episodes can contains an infinite amount of tags to provide more data about the content of the episode.

This include `podcast:person` if the episode is talking about a person (and said person is not a guest of the episode).

This also include the following tags (with some attributes examples) : 

**<podcast:book isbn="" title="" author="" url="[link to the book on any website]">**

**<podcast:game title="" developer="" publisher="" platforms="" url="">**

**<podcast:tv_show title="" director="" producer="" url="">**

**<podcast:movie title="" director="" producer="" url="">**

**<podcast:music title="" artist="" director="" producer="" url="">**

**<podcast:art title="" artist="" url="">**

**<podcast:object name="" brand="" price="" currency="" url="">**
any object, a car, a computer, a furniture, a puree press etc..

**<podcast:location name="" lat="" lng="" address="" city="" postcode="" country="">**
This duplicates #138

**<podcast:website name="" url="">**
Although we could use `<podcast:link>` from #176

**<podcast:event title="" when="(same date format as pubDate)">**
With some childrens : `podcast:location`, `podcast:website|link`

**<podcast:podcast name="" url="[website url]" rss="[rss url]">**
Can use some childrens used at channel level : `podcast:id`, `podcast:social` etc.

Some refinements are needed : 
- I'm a bit worried to set fixed attributes and be annoyed later.
- Also this could be great to settle on some kind of "id" on more tags like we have on books with ISBN.

IMHO, this would be a big added value to rss feeds. It could allow podcasters to precisely tag what they where talking about in their episode, and allow podcast players to search by topic, like YouTube does on some content :

![image](https://user-images.githubusercontent.com/469501/106633106-c3313a00-657e-11eb-9a70-38383c01db8e.png)

![image](https://user-images.githubusercontent.com/469501/106633529-389d0a80-657f-11eb-94f8-90ee41cbd162.png)

**Comment by Bigaston** [comment]
For the ID of game, maybe we can add the IGDB id or slug? I think it can work very well

**Comment by PofMagicfingers** [comment]
Yes! I guess we could use thetvdb for TV shows and imdb for movies (or any
other more "opensourcy" alternative)

**Comment by swschilke** [comment]
I would not use multiple tags that kind of express the same, a reference of or to something. Couldn't it be a "reference" tag with a value s like of book, movie, show, game etc., a description, and then a referring URL/Reference.

**Comment by PofMagicfingers** [comment]
> I would not use multiple tags

Yeah that was something I had in mind, maybe we could merge them into something like :
**<podcast:topic type="[tv|movie|music|etc]" id="[id that make sens given type]" url="[url] " [some other attr] />**

ID could be in the form of : `isbn:134432` or `tvdb:242` and such. 

We should discuss what attributes can work for all types. `artist` or `author` could work for games, TV shows, music, and movies, which are IMHO the most important/useful types. 

We could add a `date` attributes that works for release dates, events and such. 

Maybe we could move the title/name as text content but that prevent us to add children elements to be more specific or extend metadata.

**Comment by theDanielJLewis** [comment]
Whatever tags come from this should probably be in the episode metadata JSON with a master reference from the feed. Like `<podcast:metadata url="…" />`.

**Comment by PofMagicfingers** [comment]
In that case, maybe we should go for something like [JSON-LD](https://json-ld.org/) and [Schema.org](https://schema.org) for metadatas

**Comment by Bigaston** [comment]
I for and against the use of JSON metadata. For me it can be good to have it on RSS and in the same time in the JSON. Like that parser can just parse the RSS and don't have to do another request

**Comment by PofMagicfingers** [comment]
Using an external file for metadatas could be a good solution to limit bandwith usage. You get the main data in the RSS feed, and when you download or open an episode, the client download the metadata (topics, chapters, etc), only when needed.

**Comment by swschilke** [comment]
I just mentioned that I think it's a bad idea to split meta data into several files is error-prone.

**Comment by Marzal** [comment]
Hi, as per https://github.com/Podcastindex-org/podcast-namespace/discussions/749 and https://github.com/Podcastindex-org/podcast-namespace/discussions/695 we are closing old issues.

Related discussion: https://github.com/Podcastindex-org/podcast-namespace/discussions/609#discussioncomment-12543833

Please open a new [Discussions](https://github.com/Podcastindex-org/podcast-namespace/discussions) if you think that the matter it's worth pursuing.
